site/README.md

```diff
@@ -0,0 +1,54 @@
+# aip.dev static site generator
+
+Welcome to our NIH static site generator.
+
+**Note:** If you do not care about this and just want to write or modify AIPs,
+see `CONTRIBUTING.md`.
+
+## Why?
+
+GitHub Pages normally automatically builds documentation with [Jekyll][], but
+as the AIP system has grown, we are beginning to reach the limits of what
+Jekyll can handle:
+
+- AIP adoption is handled through fork-and-merge and top-down configuration
+  files will lead to repetitive merge conflicts.
+- Our grouping and listing logic has grown complicated, and had to be
+  maintained using complex and error-prone Liquid templates.
+- Jekyll is extensible but GitHub requires specific Jekyll plugins, meaning we
+  can not use off-the-shelf solutions for planned features (e.g. tabbed code
+  examples).
+- Lack of meaningful build CI caused failures.
```

Comment:
Changing the infra won't create a CI pipeline. We are still going to need to create one, and if we are still managing our workflow with GitHub, we are still going to need to create a pipeline that is triggered on merge or PR requests, runs tests, and only then allows us to merge into master/production.

So is the problem here the infrastructure/tooling, or is it that we haven't invested in a pipeline yet?

Reply:
Right. I am actually working on creating that today. :-)
However, with the old system, I am unsure how to create it at all (and I did try to figure it out).